### PR TITLE
Create klouqz.json

### PR DESCRIPTION
**Pull request by klouqzz**

### CODE REVIEW

**Comment by MaskDuck** [comment]
can you not making a new pull request thanks

**Review by MaskDuck — APPROVED** [review]
sus (wait i didnt approve)

File: domains/klouqz.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Klouqz Website's",
+  "repo": "https://github.com/klouqzcreative/klouqz.github.io",
+  "owner": {
+    "username": "Klouqz",
```
Comment (MaskDuck) [inline domains/klouqz.json:5]: uh github username pls?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by klouqzcreative** [comment]
klouqzcreative sir

MaskDuck ***@***.***>, 19 Mar 2022 Cmt, 13:33 tarihinde şunu
yazdı:

**Comment by MaskDuck** [comment]
you should edit the username field to your github username.

**Comment by MaskDuck** [comment]
also is there anything on the github pages

**Pull request closed without merging**